internal/logger/logger_setup.go

```diff
@@ -22,13 +22,13 @@ import (
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 
-	serverconfig "github.com/stacklok/minder/internal/config/server"
+	config "github.com/stacklok/minder/internal/config/server"
 )
 
 // FromFlags configures logging and returns a logger with settings matching
 // the supplied cfg.  It also performs some global initialization, because
 // that's how zerolog works.
-func FromFlags(cfg serverconfig.LoggingConfig) zerolog.Logger {
+func FromFlags(cfg config.LoggingConfig) zerolog.Logger {
```

Comment:
I renamed this import because there was only one logic `config` it could be here.  I was also (earlier) using a global.  I can take out this change if it annoys anyone.